config/gulp/file-upload.js

```diff
@@ -0,0 +1,69 @@
+require("dotenv").config();
+const { series, src } = require("gulp"),
+  vinylPaths = require("vinyl-paths"),
+  del = require("del"),
+  s3config = {
+    accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+    secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+  },
+  s3 = require("gulp-s3-upload")(s3config);
+
+/**
+ * Uploads images in /processed folder to digitalgov s3 bucket
+ * TODO: Refactor both functions into one
+ * https://github.com/uswds/uswds-compile/blob/223fe67c335135c55cf1be9dead44e6d363e219d/gulpfile.js#L204-L232
+ */
+
+function uploadImage() {
+  console.log("starting image upload");
+
+  return src("content/uploads/_working-images/processed/*")
+    .pipe(
+      s3(
+        {
+          Bucket: "digitalgov",
+          ACL: "public-read",
+        },
+        {
+          maxRetries: 5,
+        }
+      )
+    )
+    .pipe(vinylPaths(del));
+}
+
+/**
+ * Uploads files in /to-process folder to s3 static file bucket
+ * TODO: Refactor both functions into one
+ */
+
+function uploadFile() {
+  console.log("starting file upload");
+
+  return src("content/uploads/_working-files/to-process/*")
+    .pipe(
+      s3(
+        {
+          Bucket: "digitalgov/static",
+          ACL: "public-read",
+        },
+        {
+          maxRetries: 5,
+        }
+      )
+    )
+    .pipe(vinylPaths(del));
+}
```

Comment:
We'll need to keep an eye on these for performance. 

If it takes too long we can return early if directories are empty or don't exist.

Reply:
Good to know. I've run the code without any files in the `_inbox` and it doesn't run slower on my local computer at least.